Documentation/generated/CLI.md

```diff
@@ -10,6 +10,43 @@ DO NOT MODIFY IT MANUALLY, MODIFY THE SCRIPT INSTEAD.
 Here we describe Capstan CLI in detail. Please note that this very same information can be obtained
 by adding --help flag to any of the listed commands.
 
+## General:
+```
+NAME:
+   capstan - pack, ship, and run applications in light-weight VMs
+
+USAGE:
+   capstan [global options] command [command options] [arguments...]
+
+COMMANDS:
+     config            Capstan configuration
+     info              show disk image information
+     import            import an image to the local repository
+     pull              pull an image from a remote repository
+     rmi               delete an image from a repository
+     run               launch a VM. You may pass the image name as the first argument.
+     build             build an image
+     compose           compose the image from a folder or a file
+     images, i         list images
+     search            search a remote images
+     instances, I      list instances
+     stop              stop an instance
+     delete            delete an instance
+     package           package manipulation tools
+     stack, openstack  OpenStack manipulation tools
+     runtime           package runtime manipulation tools (meta/run.yaml)
+     volume            volume manipulation tools
+     help, h           Shows a list of commands or help for one command
+
+GLOBAL OPTIONS:
+   -u value                        remote repository URL (default: "https://mikelangelo-capstan.s3.amazonaws.com/")
+   --releaseTag value, --rt value  the release tag: any, latest, v0.51.0
+   --s3                            searches and downloads from S3 repository at ("https://mikelangelo-capstan.s3.amazonaws.com/")
```

Comment:
Can we get rid of this option altogether? The user can already specify an URL with the `-u` option, no? So if there's `github.com` there somewhere, we already know which repository to pull from?

Reply:
Well, I am not sure we can get rid of ```-s3```. It seems redundant, but if I have S3 repo URL defined like this in the capstan config file (```~/.capstan/config.yaml```):
```
CAPSTAN_REPO_URL: https://mikelangelo-capstan.s3.amazonaws.com/
```
how do I tell capstan in all the relevant command cases to pull/use packages or loader from S# repo instead of github? I could simply recognize the act of passing the -u, but -u merely overrides what is in config.

So in essence -s3 acts as a switch between 'github' mode and 'S3' mode.